src/main/java/core/basesyntax/operation/BalanceOperation.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.operation;
+
+import java.util.Map;
+
+public class BalanceOperation implements OperationHandler {
+    @Override
+    public void execute(Map.Entry<String, Integer> entry, int value) {
+        entry.setValue(value);
+    }
+
+    public static class ReturnOperation implements OperationHandler {
```

Comment:
The nested `ReturnOperation` class should be in its own file as well, especially since it implements the `OperationHandler` interface, which indicates it serves a distinct role.